fbfmaproom/fbfmaproom.py

```diff
@@ -200,6 +200,17 @@ def format_enso(x):
         return "El Niño"
     assert False, f"Unknown enso state {x}"
 
+def format_nma_bady(x):
```

Comment:
NMA is the National Meteorology Agency of Ethiopia. We shouldn't have anything country-specific in the code, all differences between countries are supposed to be in the config file. So either we add a mechanism for defining custom categories in the config file, or we call this "format_bad_ternary" (as opposed to format_bad which is for binary classification) and let it be used for other countries too.

Reply:
BTW yes, I think you found the right mechanism for adding new categories.